dual_net.py

```diff
@@ -247,9 +258,15 @@ def host_call_fn(policy_output, value_output, pi_tensor, policy_cost,
         }
         # Create summary ops so that they show up in SUMMARIES collection
         # That way, they get logged automatically during training
-        for metric_name, metric_op in metric_ops.items():
-            tf.summary.scalar(metric_name, metric_op[1])
-        return metric_ops
+        if FLAGS.use_tpu:
+            with summary.create_file_writer(FLAGS.model_dir).as_default(), summary.record_summaries_every_n_global_steps(FLAGS.iterations_per_loop * 2):
```

Comment:
FWIW, the estimator model_fn interface allows config/params as args, which will get passed through. This would be the official way to do it sans FLAGS.